styles.css

```diff
@@ -263,8 +307,13 @@ span {
     height: auto;
   }
 
+  .dark-desktop {
+    background: #2e2e2e !important;
+    color: #aeaeae;
+  }
+
   .mobile-only {
-    display: none;
+    display: none !important;
   }
 }
 
```

Comment:
Kindly include adequate styling for the desktop version (and mobile version too) of the project in order to correspond to the design guideline.

Your design:
![image](https://user-images.githubusercontent.com/20554583/193436448-3a331733-0aa9-491c-83e8-484bc998d336.png)
![image](https://user-images.githubusercontent.com/20554583/193436520-7db2858d-c5e6-4b64-ae24-147366d4b61e.png)
![image](https://user-images.githubusercontent.com/20554583/193436527-0d12476e-92b5-48ce-9142-c5e21751b116.png)
![image](https://user-images.githubusercontent.com/20554583/193436535-bf611dff-e4a7-4047-8916-e2b0d179f18b.png)
![image](https://user-images.githubusercontent.com/20554583/193436544-9d9a0f84-81bf-475d-89ed-91bb093071e5.png)


Design guideline:
![image](https://user-images.githubusercontent.com/20554583/193436469-860c5da1-663d-42ee-b33f-da03ec0b84a1.png)
![image](https://user-images.githubusercontent.com/20554583/193436474-0fac79bd-bdbf-4c71-ad36-80370e20673d.png)
![image](https://user-images.githubusercontent.com/20554583/193436484-203dcf25-bd1a-4991-a294-610a39646bec.png)
![image](https://user-images.githubusercontent.com/20554583/193436495-2b88c2b2-8db0-43ee-a9eb-29f529903bb6.png)

Reply:
It was clearly said that Bootstrap had to be installed to display the proyect correctly, I even said this in the submition.

Reply:
Hi @Swarzstein,

That was a costly oversight. Sorry about that.